### PR TITLE
added ExASLM case study

### PR DESCRIPTION
**Pull request by mehr0096**

### CODE REVIEW

**Comment by AdvancedImagingUTSW** [comment]
I don't think you should have spaces in your filename...

**Comment by zacsimile** [comment]
This is an `.rtf`. Sphinx requires an `.rst` (re-structured text).

**Comment by AdvancedImagingUTSW** [comment]
You should copy the formatting already established for consistency. See here:

https://github.com/TheDeanLab/navigate/blob/develop/docs/source/case_studies/acquire_mesospimbt.rst?plain=1

File: docs/source/case_studies/acquire_exASLM.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+This is a case study in using the software to image with a `Expansion ASLM Microscope`.
+
+Setting the beam parameters
+---------------------------
+
+Make sure the imaging chamber is empty or, if a sample is mounted, the sample is not in the beam path.
+
+#. Select "Continuous Scan" from the dropdown next to the :guilabel:`Acquire` button. Press :guilabel:`Acquire`. This will launch a live acquisition mode.
+#. Go to the :guilabel:`Channels` tab. Choose the wavelength you want to align. Set the laser's :guilabel:`Power` to `100.0`. Change :guilabel:`Filter` to an "Empty" option.
+#. Go to the :menuselection:`Microscope Configuration --> Waveform Parameters`. A popup named :guilabel:`Waveform Parameter Settings` will appear. Make sure the :guilabel:`Mode` matches "mesoSPIM BT" and the :guilabel:`Magnification` matches the magnification of the objective you are using.
```
Comment (zacsimile) [inline docs/source/case_studies/acquire_exASLM.rst]: It still says "mesoSPIM BT" here.

File: docs/source/case_studies/acquire_exASLM.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+#. Select "Continuous Scan" from the dropdown next to the :guilabel:`Acquire` button. Press :guilabel:`Acquire`. This will launch a live acquisition mode.
+#. Using the :guilabel:`Stage Control`, go to a shallow Z-position in the sample. Under the :guilabel:`Channels` tab, in :guilabel:`Stack Acquistion Settings (um)` press :guilabel:`Set Start Pos`.
+#. Go to a deep Z-position in the sample. Press :guilabel:`Set End Pos`.
+#. Make sure :guilabel:`Set Foc` is 0 for both the :guilabel:`Set Start Pos` and :guilabel:`End Pos`.
+#. Type the desired step size (units um) in the :guilabel:`Step Size` dialog box  :guilabel:`Stack Acquistion Settings (um)`. Step size can only be increments of 0.1 (um) and the minimum is 0.2 (um).
+#. Select the number of color channels needed imaging in the :guilabel:`Channel tab` under :guilabel: `Channel Settings`. Select the correct filter for each channel by using the dropdown menu after each channel under the :guilabel:`Filter`.
+#. Change the exposure time by changing number in the :guilabel:`Exp. Time (ms)` for each channels. For the ORCA Lightning camera using ASLM mode, the minimum frame rate is 75 ms and the maximum is 100 ms.
```
Comment (zacsimile) [inline docs/source/case_studies/acquire_exASLM.rst:44]: Only 75-100 ms? We should fix that.

File: docs/source/case_studies/acquire_exASLM.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+#. Select "Continuous Scan" from the dropdown next to the :guilabel:`Acquire` button. Press :guilabel:`Acquire`. This will launch a live acquisition mode.
+#. Go to the :guilabel:`Channels` tab. Choose the wavelength you want to align. Set the laser's :guilabel:`Power` to `100.0`. Change :guilabel:`Filter` to an "Empty" option.
+#. Go to the :menuselection:`Microscope Configuration --> Waveform Parameters`. A popup named :guilabel:`Waveform Parameter Settings` will appear. Make sure the :guilabel:`Mode` matches "mesoSPIM BT" and the :guilabel:`Magnification` matches the magnification of the objective you are using.
+#. :guilabel:`Galvo 0` digitally sweeps the beam across the field of view. To align the axially-swept light sheet parameters, set the :guilabel:`Galvo 0` :guilabel:`Amplitude` to `0.0`. Set the :guilabel:`Offset` so that the beam is centered in the field of view. Click the :guilabel:`Camera View` to toggle the crosshairs, which indicate the center of the field of view. Adjust the :guilabel:`F` (focus) value in the :guilabel:`Stage Control` panel until the beam is as thin/focused as possible.
```
Comment (zacsimile) [inline docs/source/case_studies/acquire_exASLM.rst]: This is not true for the upright...

File: docs/source/case_studies/acquire_exASLM.rst (hunk 1 above)
Comment (mehr0096) [reply docs/source/case_studies/acquire_exASLM.rst]: fixed

File: docs/source/case_studies/acquire_exASLM.rst (hunk 3 above)
Comment (mehr0096) [reply docs/source/case_studies/acquire_exASLM.rst]: removed

File: docs/source/case_studies/acquire_exASLM.rst (hunk 2 above)
Comment (mehr0096) [reply docs/source/case_studies/acquire_exASLM.rst:44]: These parameters are for ASLM imaging. It is theoretically possible to go below 75 ms but we found that beam isn't straight at lower exposure times. This was tested with the older tube lens and not with the higher magnification tube lens that we are using now.

File: docs/source/case_studies/acquire_exASLM.rst [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+#. Once the beam is straight, slowly change the wavelength's :guilabel:`Offset` until the beam has an even thickness across the field of view.
+#. Under :guilabel:`Waveform Parameter Settings`, press :guilabel:`Save Configuration`.
+#. Under the :guilabel:`Channels` tab, restore the filter to its non-empty position.
+#. Go to :guilabel:`Camera Settings` and ensure that "Light-Sheet" is selected under :guilabel:`Sensor Mode`. Slowly increase the wavelength's :guilabel:`Amplitude` until the beam becomes a straight line across the screen. If the beam does not become straighter, try changing the camera's :guilabel:`Readout Direction`.
+#. If beam is diverges towards the ends of the camera, reduce the :guilabel:`Height` of camera under :guilabel:`Region of Interest Settings` tab by 10 pixels.
+#. After optimal parameters are found, swap spherical lens for cylindrical lens
```
Comment (zacsimile) [inline docs/source/case_studies/acquire_exASLM.rst]: Add a period.

File: docs/source/case_studies/acquire_exASLM.rst [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+#. Change the exposure time by changing number in the :guilabel:`Exp. Time (ms)` for each channels. For the ORCA Lightning camera using ASLM mode, the minimum frame rate is 75 ms and the maximum is 100 ms.
+#. Set :guilabel:`Interval` to be 1.0 for each channel.
+#. Set :guilabel:`Defocus` to be 0 for each channel.
+#. Select "Constant Velocity Acquisition" from the dropdown next to the :guilabel:`Acquire` button. Press :guilabel:`Acquire`.
+#. Enter the sample parameters in the :guilabel:`File Saving Dialog` that pops up. Make sure to save to SSD drive or change buffer size in configuration file to prevent any overwriting of images. Then Press :guilabel:`Acquire Data`.
+#. To change frame buffer size, in the :guilabel:`experiment.yaml` file in your local navigate directory in the :guilabel:`config` folder under the :guilabel:`CameraParameters` section, change :guilabel:`databuffer_size` to desired number of frames. Make sure the size of the desired number of frames isn't above the available RAM in the computer.
```
Comment (zacsimile) [inline docs/source/case_studies/acquire_exASLM.rst]: Feels like this last one is a note, not part of the ordered list. Do you change the buffer size often?

**Pull request merged**